xwiki-platform-core/xwiki-platform-ldap/xwiki-platform-ldap-authenticator/src/main/java/com/xpn/xwiki/plugin/ldap/XWikiLDAPConnection.java

```diff
@@ -321,13 +321,17 @@ public boolean checkPassword(String userDN, String password, String passwordFiel
 
             ldapToXWikiAttribute(searchAttributeList, attributeSet);
         } catch (LDAPException e) {
-            LOGGER.debug("LDAP Search failed", e);
+            if(LOGGER.isInfoEnabled()) {
```

Comment:
IMO the "if" should be located inside the detailledLDAPExceptionLog() method since the JVM will optimize the call anyway it won't cost more and it's more legible.